bin/sidecli.re

```diff
@@ -245,9 +245,10 @@ let info_sign_block = {
   let doc = "Sign a block hash and broadcast to the network manually, useful when the chain is stale.";
   Term.info("sign-block", ~version="%‌%VERSION%%", ~doc, ~exits, ~man);
 };
-let sign_block = (wallet_file, block_hash) => {
-  let.await wallet = Files.Wallet.read(~file=wallet_file);
-  let signature = Signature.sign(~key=wallet.priv_key, block_hash);
+let sign_block = (folder, block_hash) => {
+  let file = folder ++ "/identity.json";
```

Comment:
Are the standard names for files like this included in the files api? if so can we refer to that instead of hardcoding them in multiple places? not urgent

Reply:
That's the idea and why I'm making this changes, in a future PR to put them on the Files module